views/admin/util/topNav.cfm

```diff
@@ -1,6 +1,6 @@
 <cfscript>
 	topLevelNav       = renderView( view="/admin/util/topNavItems" );
-	userMenu          = renderView( view="/admin/util/userMenu", cache=true, cacheSuffix=event.getAdminUserId() );
+	userMenu          = renderView( view="/admin/util/userMenu" );
```

Comment:
Going to get some merge conflict fun here - or not. Need to be careful with changes in https://github.com/pixl8/preside-ext-alt-admin-theme/pull/2 that will need taking into consideration here also.

Reply:
Actually, I wonder whether we can take another approach here? Seems like we may be adding a bunch of page load here where we don't need to.

Could we:

1. output a placeholder for the homepage location when rendering the user menu, e.g. `{{userhomepage}}`
2. replace this after rendering them menu, e.g. (pseudocode):

```cfc
userMenu = renderView( ... ); // unchanged

userMenu = Replace( userMenu, "{{userhomepage}}", dynamicallyCalculatedUserHomepageLink, "all" );
```